lib/inat_vision_api.py

```diff
@@ -77,6 +78,14 @@ def h3_04_bounds_route(self):
             return f"Unknown taxon_id {taxon_id}", 422
         return results_dict
 
+    def build_info_route(self):
+        return {
+            "git_branch": os.environ["GIT_BRANCH"],
```

Comment:
This will raise an error if these environment variables are not defined. Maybe that's desired. Some of the other versions will handle that a little better by returning empty hashes or nothing at all. The node API for example will also return a 500 if this endpoint fails, so in deployments where these environment variables are not defined, a handful of the build_info endpoints in the different codebases will render a 500 error of some sort

Reply:
I replaced them by os.getenv("xxx", "") to avoid such failure